### DISCUSSION TITLE
Quickbooks online

### DISCUSSION THREAD

**Discussion by nwns** [inventree/InvenTree#3318] [Q&A] [answered]
I have seen some previous mention of Quickbooks Online in a couple of issues, but it is unclear if anything came of those discussions. Is there known existing (completed, in progress, or partial) integration for QBO and Inventree.

**Comment by matmair** [comment, marked as answer]
@nwns to my knowledge there are at least two private integrations with quickbooks.
All solutions I know of are made specific for the user base and not targeted for public release.
Depending on what you want this can be a multi-month development story. Good separation of tasks between the systems and a clear scope of development can help to reduce the effort.

**Reply by matmair** [reply]
@nwns  as there was no further interaction I will mark this as the answer. If you require further information feel free to reach out.
